src/wallet/wallet.h

```diff
@@ -961,6 +961,8 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     //! Get the LegacyScriptPubKeyMan which is used for all types, internal, and external.
     LegacyScriptPubKeyMan* GetLegacyScriptPubKeyMan() const;
     LegacyScriptPubKeyMan* GetOrCreateLegacyScriptPubKeyMan();
+    LegacyDataSPKM* GetLegacyDataSPKM() const;
+    LegacyDataSPKM* GetOrCreateLegacyDataSPKM();
 
     //! Make a LegacyScriptPubKeyMan and set it for all types, internal, and external.
```

Comment:
The function still creates a `LegacyScriptPubKeyMan` in normal operation.

Reply:
`LegacyScriptPubKeyMan` is a `LegacyDataSPKM`, so the comment would be more correct, but it could be corrected later when removing `LegacyScriptPubKeyMan` - unless you plan to rename `LegacyDataSPKM` back into `LegacyScriptPubKeyMan` once the deletion of the latter is complete.

Reply:
I think the meaning is well understood, both before and after `LegacyScriptPubKeyMan` is deleted. `SPKM` is an abbreviation for `ScriptPubKeyMan` anyways.